qmlcomps/videocamera/Joystick.qml

```diff
@@ -80,23 +83,30 @@ Item {
     MouseArea {
```

Comment:
This can use a `DragHandler` and `WheelHandler`?

Reply:
Mouse handler was already implemented so i just get wheel event